setup.py

```diff
@@ -68,8 +63,8 @@ def _package_doc(name):
         },
       entry_points="""
       [zope2.initialize]
-      Products.%s = Products.%s:initialize
+      Products.{} = Products.{}:initialize
       [distutils.commands]
       ftest = zope.testrunner.eggsupport:ftest
-      """ % (NAME, NAME),
+      """.format(NAME, NAME),
```

Comment:
```suggestion
      """,
```

(Sorry for three suggestions, it didn't let me select and change the whole block because it contains deleted lines)